scripts/build-wasm.sh

```diff
@@ -4,12 +4,14 @@ set -euo pipefail
 rm -rf rust/kcl-wasm-lib/pkg
 mkdir -p rust/kcl-wasm-lib/pkg
 rm -rf rust/kcl-lib/bindings
+rm -rf rust/kcl-api/bindings
 
 cd rust
 export RUSTFLAGS='--cfg getrandom_backend="wasm_js"'
 wasm-pack build kcl-wasm-lib --release --target web --out-dir pkg
 export RUSTFLAGS=''
 cargo test -p kcl-lib --features artifact-graph export_bindings
+cargo test -p kcl-api export_bindings
```

Comment:
Do we need to do this for `kcl-error` also? I guess I'm confused how things are working at all now if we do need it.